templates/zerver/help/index.md

```diff
@@ -152,3 +152,7 @@ as an **organization**.
 ## Linking to a Zulip server
 
 * [Join Zulip chat badge](/help/join-zulip-chat-badge)
+
+# Analytics
+
+* [Analytics](/help/analytics)
```

Comment:
ok, i thought that a separate heading might be useful for the future i.e. when more features are added to analytics